templates/prescription/general_edit.html

```diff
@@ -256,10 +265,24 @@
 {$ENDING_JAVASCRIPT}
 </script>
 </form>
+<!-- Has to be loaded after the form -->
+<script type="text/javascript" src="{php} echo $GLOBALS['assets_static_relative']; {/php}/jquery-min-1-9-1/index.js"></script>
+<script type="text/javascript" src="{php} echo $GLOBALS['assets_static_relative']; {/php}/jquery-ui-1-12-1/jquery-ui.js"></script>
+
 {literal}
 <!-- for the fancy jQuery stuff -->
 <script type="text/javascript">
 
+$(function() {
+    
+    //Drug form autocomplete to text box.
+    $(".wDrug").autocomplete({
```

Comment:
If this function is specific to weno rx, then would only output this if he weno rx feature is on.

Reply:
Ok, will do.

Reply:
I don't know how to do this. I have tried <?php if ($GLOBALS['weno_rx_enable']) ?> and
 
{php} if ($GLOBALS['weno_rx_enable']) { {/php}

{php} } {/php}

I think because of the {literal} function no matter what is typed in there will be output.